packages/cli/src/modules/dynamic-credentials.ee/workflow-status.controller.ts

```diff
@@ -37,14 +39,16 @@ export class WorkflowStatusController {
 
 		const isReady = status.every((s) => s.status === 'configured');
 
+		const basePath = this.urlService.getInstanceBaseUrl();
+
 		const executionStatus: WorkflowExecutionStatus = {
 			workflowId,
 			readyToExecute: isReady,
 			credentials: status.map((s) => ({
 				credentialId: s.credentialId,
 				credentialStatus: s.status,
 				credentialType: s.credentialType,
-				authorizationUrl: `/credentials/${s.credentialId}/authorize?resolverId=${encodeURIComponent(s.resolverId)}`,
+				authorizationUrl: `${basePath}/rest/credentials/${s.credentialId}/authorize?resolverId=${encodeURIComponent(s.resolverId)}`,
```

Comment:
I think we should use this: https://github.com/n8n-io/n8n/blob/master/packages/@n8n/config/src/configs/endpoints.config.ts#L93

Reply:
Very good point. Thank you 👍